### PR TITLE
Milestone 12: Validate contact form 🧾🔍

### PR DESCRIPTION
**Pull request by mdg0410**
In this milestone:

- Work pair programming
- Add validateForm.js file
- Validate `input name` , `input email` and `input text` with `JS` file
- Add validateForm.css file
- Use of validateForm.css file to style elements inside the `form` tag

### CODE REVIEW

**Review by KanzaTahreem — COMMENTED** [review]
# INVALID ❌ 

Hi @mdg0410  👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
You have submitted a project that is a pair programming task, but I cannot see the commits from your coding partner. 

image

Please make sure that for a pair programming task, everybody has contributed code on each project.

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/validateForm.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    case 'name':
+      checkName();
+      break;
+    case 'message':
+      checkText();
+      break;
+    default:
+      break;
+  }
+}));
```
Comment (GzimAsani) [inline js/validateForm.js]: - [ ] Currently, you do not have any validation for email. The user can submit an email in uppercase which should not be allowed according to the requirement. Also the user can submit an email without the `@` sign which should not be allowed as well

Comment (GzimAsani) [reply js/validateForm.js]: - [ ] You have an error that comes from Card.js file kindly check that 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello TEAM                  :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the form so far :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/validateForm.js (hunk 1 above)
Comment (mdg0410) [reply js/validateForm.js]: ## Hi @GzimAsani 🖐

*Kindly accept my recommendation*

You should have read the initial comment where I clarified that I validated the email with HTML5 to make good use of the reading material provided by Microverse.

imagen

Kindly remember to change the name of the person to whom you are making the revision.

imagen

Comment (GzimAsani) [reply js/validateForm.js]: - [ ] Hi @mdg0410, kindly check project requirements, the form should not be able to submit when error, and yours do submit. The error message should show around the button get in touch, which yours doesn't 
image

Comment (mdg0410) [reply js/validateForm.js]: # Thanks @GzimAsani 😎🤘,
If it wasn't for your comment I wouldn't have been motivated to improve my code and put more conscience in checking better the requirements.

Good luck and thanks again. 🍀

Comment (GzimAsani) [reply js/validateForm.js]: > # Thanks @GzimAsani 😎🤘,
> If it wasn't for your comment I wouldn't have been motivated to improve my code and put more conscience in checking better the future requirements.
> 
> Good luck and thanks again. 🍀

I am glad that I could help and I am sorry if I wasn’t clear enough the first time 🙏

Comment (mdg0410) [reply js/validateForm.js]: Do not worry 👽

**Comment by mdg0410** [comment]
> Correction 

***Validating input `email` with javascript***

|Correct|Incorrect|
|:-------:|:--------:|
|imagen|imagen|

> Correction

***Restructuring the `card.js` file***

imagen

**Review by nahnahdev — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Nice Design :heavy_check_mark: 
- [ ] All validations working as expected :heavy_check_mark: 

## Optional suggestions
- N / A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**